src/main/resources/reference.conf

```diff
@@ -56,6 +63,67 @@ jibri {
     audio-source = "alsa"
     // The audio device that will be used to capture audio on Linux
     audio-device = "plug:bsnoop"
+
+    // The path to the ffmpeg binary.
+    ffmpeg-binary = "ffmpeg"
+    ffmpeg = [ ${jibri.ffmpeg.ffmpeg-binary}, "-y", "-v", "info" ]
+    // Input parameters specific to linux
+    input-linux = [
+      "-f", "x11grab"
+      "-draw_mouse", "0"
+      "-r", ${jibri.ffmpeg.framerate}
+      "-s", ${jibri.ffmpeg.resolution}
+      "-thread_queue_size", ${jibri.ffmpeg.queue-size}
+      "-i", ":0.0+0,0"
+      "-f", ${jibri.ffmpeg.audio-source}
+      "-thread_queue_size", ${jibri.ffmpeg.queue-size}
```

Comment:
You have thread_queue_size listed twice

Reply:
Yes, that's for the different sources (-i), it's been like this in jibri:
https://github.com/jitsi/jibri/blob/0271286b4c9dfbe5807a835bd1244e919be47f39/src/main/kotlin/org/jitsi/jibri/capture/ffmpeg/Commands.kt#L28C9-L28C35